packages/sdk/src/sdk/sdk.deposit.ts

```diff
@@ -126,11 +128,6 @@ export async function getDepositData<
       config.source.weight,
       moonChain,
     ),
-    // xcmFeeBalance
-    foreignPolkadot.getGenericBalance(
```

Comment:
IS this being removed because it's a pain to make it work with eq?

Reply:
We discussed this with Mario in the comments. The issue was that for DEV fee was in 10 decimals and the balance here was in 18 decimals. I checked in the dapp we were not using it so the easiest solution is to just remove it. I don't think that anybody else is using it either.

Reply:
Shouldn´t we use it to check if the user has enough balance to pay for the XCM fee?

Reply:
No, we are using `sourceBalance` or `sourceFeeBalance`.